bitcoin/Cargo.toml

```diff
@@ -21,6 +21,7 @@ secp-lowmemory = ["secp256k1/lowmemory"]
 secp-recovery = ["secp256k1/recovery"]
 bitcoinconsensus-std = ["bitcoinconsensus/std", "std"]
 
+
```

Comment:
This shouldn't really be in here but I won't hold you up for it :)

Reply:
Doh, will fix